Tornado port

stefanv opened the pull request.

stefanv committed:
Rename app server

stefanv committed:
Port web server to Tornado

bnaul committed:
Add featurize handler

stefanv committed:
Add model handler

stefanv committed:
Add prediction and plot handlers

stefanv committed:
Remove flask-isisms

stefanv committed:
Use new location of check_model_param_types

bnaul commented on cesium_app/handlers/task.py:
```diff
@@ -0,0 +1,57 @@
+# TODO pass in user somehow
+# TODO show error notification for failures
+@app.route("/task_complete", methods=['POST'])
+def task_complete():
```
This file can be deleted, right?

stefanv committed:
Remove old task file

bnaul commented:
What about the `task_server`? If that's sticking around, we could prune the logic that's no longer used. Otherwise 👍  pending tests!

stefanv committed:
Factor out host/port for test driver

stefanv committed:
Port changes to master

stefanv committed:
Update tests to match new error messages

stefanv committed:
Update installation instructions; db fixes

stefanv committed:
Use h5netcdf engine to export netcdf files

stefanv committed:
Revert change to port 9000

bnaul committed:
Restore zero-features check

bnaul committed:
Fix name conflict in prediction handler

bnaul committed:
Specify netCDF engine in config file, refactor test data

stefanv committed:
Merge pull request #2 from bnaul/stefanv-tornado_port

Specify netCDF engine in config file, refactor test data

stefanv committed:
Remove joblib from dependencies

We use the version in scikit-learn

stefanv committed:
Use specified package name when checking dependencies

Previously, the lowercase name was used.  I'm not sure why, so now we're just
using the package name as it is specified in `requirements.txt`.

acrellin commented:
Which test got removed?

bnaul commented:
What do you mean?

bnaul commented:
Oh did the total # change? Could have been a merge conflict mistake or something...

acrellin commented:
There were 21 tests running on my PR that just got merged, and 20 here (from the Travis logs).

acrellin commented:
Yeah, exactly.

bnaul commented:
Ah it's because `check_model_param_types` was removed.

acrellin commented:
Ah, okay. Thanks for clearing that up.
